modules/nf-core/nonpareil/set/tests/main.nf.test

```diff
@@ -0,0 +1,52 @@
+nextflow_process {
+
+    name "Test Process NONPAREIL_SET"
+    script "../main.nf"
+    process "NONPAREIL_SET"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "nonpareil"
+    tag "nonpareil/set"
+
+    test("candidatus_portiera_aleyrodidarum") {
+
+
+        setup {
+            run("NONPAREIL_NONPAREIL") {
+                script "../../../nonpareil/nonpareil/main.nf"
+                process {
+                    """
+                    input[0] = Channel.fromList([
+                        [[ id:'test', single_end:false ], file(params.test_data['candidatus_portiera_aleyrodidarum']['illumina']['test_1_fastq_gz'], checkIfExists: true)],
+                        [[ id:'test2', single_end:false ], file(params.test_data['candidatus_portiera_aleyrodidarum']['illumina']['test_1_fastq_gz'], checkIfExists: true)],
+                    ])
+                    input[1] = 'fastq'
+                    input[2] = 'kmer'
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = NONPAREIL_NONPAREIL.out.npo.map{meta, npo -> [[id:'test'], npo]}.groupTuple()
```

Comment:
What is this part for?

Reply:
SET plots multiple curves (npo) files on one plot, so I need to aggregated them together into a list (the `id:test` is unnecessary though tbh)